src/main/kotlin/step2/domain/Lotto.kt

```diff
@@ -0,0 +1,19 @@
+package step2.domain
+
+data class Lotto(var numbers: List<Int>) {
```

Comment:
```suggestion
data class Lotto(val numbers: List<Int>) {
```
val이 되어도 될 것 같아요! 코틀린은 기본으로 불변을 제공해주기 때문에 꼭 필요한 경우가 아니라면 언어적 이점을 살리기 위해서 불변 자료형, val 키워드를 사용하는 것을 지향하는게 좋습니다. 저는 어떤 프로퍼티나 자료형을 만들 때 항상 불변, 읽기 전용으로 만들어두고 나중에 값 조작이 발생하는 일이 생기면 그때 수정하곤 합니다.